.github/workflows/build.yml

```diff
@@ -5,10 +5,11 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
GitHub caches commonly used images and workflows based on them execute very fast, this bit me once when every Debian workflow I tried to run took over an hour for an open source repo, please check if an older version of ubuntu is cached so this doesn't suprise us.